### PR TITLE
Add superset

### PR DESCRIPTION
**Pull request by v9n**
First version of superset with docker compose.

### CODE REVIEW

File: docker-compose-ss.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  - SUPERSET_ENV=production
+  - SUPERSET_LOAD_EXAMPLES=no
+  - SUPERSET_SECRET_KEY=${SUPERSET_SECRET_KEY:-secretkey}
+  - CYPRESS_CONFIG=false
+  - SUPERSET_PORT=8088
+  - MAPBOX_API_KEY=${MAPBOX_API_KEY:-}
+
+version: "3.9"
+services:
+  redis:
```
Comment (chrisli30) [inline docker-compose-ss.yml:34]: oh superset depends on and leverage redis? The componet setup is getting complex.

File: docker/superset/docker-frontend.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+set -e
+
+# Packages needed for puppeteer:
+apt update
+apt install -y chromium
```
Comment (chrisli30) [inline docker/superset/docker-frontend.sh:22]: The frontend also depends on chromium, hmm.

**Review by chrisli30 — APPROVED** [review]
Looks good. These are lots of setup.

File: docker-compose-ss.yml (hunk 1 above)
Comment (v9n) [reply docker-compose-ss.yml:34]: yeah, it has quite a bit of component due to need to implement multiple ting such as auto refresh data, send out notification of query result etc. so it has a lot of cache storing in redis.

though we hide all of these behind docker compose.

File: docker/superset/docker-frontend.sh (hunk 2 above)
Comment (v9n) [reply docker/superset/docker-frontend.sh:22]: It only uses when we want to send out notification that has pictures of the chart. It used chromium to render the screenshot of the chart so I can embed the screenshot into the email. If we don't use it, it won't run chromium.

Comment (chrisli30) [reply docker/superset/docker-frontend.sh:22]: Cool, gotcha. Let’s keep it as-is for now.

**Pull request merged**